setup.py

```diff
@@ -56,6 +60,71 @@ def run(self):
         bin_path = os.path.join(self.install_lib, 'pact', 'bin')
         os.mkdir(bin_path)
         install_ruby_app(bin_path)
+        install_rust_app(bin_path)
+
+
+def install_rust_app(bin_path):
+    """
+    Download the relevant rust binaries and install it for use.
+
+    :param bin_path: The path where binaries should be installed.
+    """
+    target_platform = platform.platform().lower()
+
+    if 'darwin' in target_platform or 'macos' in target_platform:
+        suffix = 'libpact_ffi-osx-x86_64'
+    elif 'linux' in target_platform:
+        suffix = 'libpact_ffi-linux-x86_64'
+    elif 'windows' in target_platform:
+        suffix = 'pact_ffi-windows-x86_64'
+    else:
+        msg = ('Unfortunately, {} is not a supported platform. Only Linux,'
+               ' Windows, and OSX are currently supported.').format(
+            platform.platform())
+        raise Exception(msg)
+
+    if 'windows' in platform.platform().lower():
+        fetch_lib(bin_path, suffix, 'dll')
+        fetch_lib(bin_path, suffix, 'dll.lib')
+        fetch_lib(bin_path, suffix, 'lib')
```

Comment:
If you are using the DLL, you don't need the static lib (.lib). But there is no harm in downloading it, apart from it being the biggest one.